pkg/osbuild/skopeo_source_test.go

```diff
@@ -33,27 +52,30 @@ func TestNewSkopeoSource(t *testing.T) {
 	assert.Nil(t, item.Image.TLSVerify)
 
 	// empty name
-	assert.Panics(t, func() {
+	expectedErr := regexp.MustCompile(`source item osbuild.SkopeoSourceItem.* has empty name`)
```

Comment:
Since this is needed every time I'd just pass the regex as an argument and let the function compile it.